testbed/addons/flux/construct.ts

```diff
@@ -0,0 +1,85 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+import { KubernetesManifest } from '@aws-cdk/aws-eks';
+
+export interface FluxProps extends cdk.StackProps {
+    cluster: eks.Cluster;
+    repositories: Repository[];
+}
+
+export interface RepositoryOptions {
```

Comment:
Let's pick a consistent term `Options` or `Props`

Reply:
props it is.